### PR TITLE
modal add event finished

### PR DESCRIPTION
**Pull request by dsusviela**
Modified:

Gather.js
 - Added all references for the new modal

CreatePocketModal.js:
  - Removed unnecessary import

localization/en.js and localization/es.js.
  - Added several new strings for the new modal

New:
GatherAddEventModal.js

Please note that I ignore what is the file that has the [m at every end. This was generated at some point. To test the modal, you might need to change the default corresponding value of Gather.js state. @frabago will make the modal appear, he is handling the opening and closing from the map. Please note that having several modals open makes them appear erratically as you can read at react-native-modal documentation, for instance when testing myself, i could see the confirm exit modal appear when it shouldnt. These issues shouldnt be on master as the code implemented here does not affect it.

### CODE REVIEW

File:  [hunk 1]
```diff
@@ -0,0 +1,35 @@
+  EndTravelButton[m
```
Comment (frabago) [inline ]: This file should be added to test the modal?

Comment (dsusviela) [reply ]: Nope, i dont think so. I dont really know where it came from

**Comment by dsusviela** [comment]
Aight im having a problem with react native. Maybe im too dumb to realize what im doing wrong but the if isnt working at ALL.

**Comment by dsusviela** [comment]
Silly me was passing an instruction not a function to the onPress of the button. Should be working now

**Pull request merged**